File: RepoTransBench/target_projects/Java/Go/mohak1712_UberUX/public_tests/password_activity_public_test.go
package public_tests

import (
	"testing"
	"mohak1712_uberux/tests"
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/mock"
)

func TestPasswordActivityOnCreateExecutesWithoutCrashPublic(t *testing.T) {
	p := new(tests.PasswordActivity)
	p.On("OnCreate", mock.Anything).Return()
	bundle := tests.Bundle{"public_test_char": "P"}
	p.OnCreate(bundle)
	assert.NotNil(t, p)
}

File: RepoTransBench/target_projects/Java/C#/mohak1712_UberUX/public_tests/PasswordActivityPublicTest.cs
using Moq;
using Xunit;

namespace UberUX.Tests.Public
{
    public class PasswordActivityPublicTest
    {
        private UberUX.PasswordActivity activity;

        public PasswordActivityPublicTest()
        {
            var mock = new Mock<UberUX.PasswordActivity> { CallBase = true };
            activity = mock.Object;
        }

        [Fact]
        public void TestOnCreateExecutesWithoutCrash_Public()
        {
            var bundle = new System.Collections.Generic.Dictionary<string, object>();
            bundle["public_test_char"] = 'P';
            activity.onCreate(bundle);
            Assert.NotNull(activity);
        }
    }
}